drivers/hmis/app/graphql/types/hmis_schema/service_type_input.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class HmisSchema::ServiceTypeInput < BaseInputObject
+    description 'Create service type input'
+
+    argument :name, String, required: true
+    argument :custom_service_category_id, ID, required: true
+    argument :hud_record_type, ID, required: false
+    argument :hud_type_provided, ID, required: false
```

Comment:
Oh I forgot to mention these, we don't need them. The HUD service types are fixed, so users will only be managing custom services.

Reply:
Just confirming, we do still need `supports_bulk_assignment`, correct?

Reply:
yep!